zulu/datetime.py

```diff
@@ -379,6 +382,83 @@ def replace(self,
 
         return DateTime(*args)
 
+    def span(self, frame, count=1):
+        """Returns two new :class:`Datetime <datetime.datetime>` objects,
+        representing the time span of the :class:`Datetime <datetime.datetime>`
+         object in a given time frame.
+
+        Args:
+             frame (str): A time frame. Ex: year, month, week, day, etc.
+             count (int): Number of frames to span
+
+        Returns:
+            floor: <datetime.datetime>
+            ceil: <datetime.datetime>
+        """
+        abs_name, rel_name, rel_steps = self.get_time_frames(frame)
+
+        index = self.ATTRS.index(abs_name)
+        frames = self.ATTRS[:index + 1]
+
+        values = [getattr(self, f) for f in frames]
+
+        # year, month, day are all required by zulu.Datetime. If the time
+        # frame given is 'year' then we fill the other two arguments 'day'
+        # and 'month' with the value 1. For example, if the given statement is
+        # zulu.now().span('year') then the values will be [2016, 1, 1]. So
+        # when these values are sent to the class method we would get a
+        # zulu.datetime.datetime object
+        for i in range(3 - len(values)):
+            values.append(1)
+
+        # To maintain consistency, we will return both floor and ceil values
+        # in datetime.datetime format.
+        floor = self.__class__(*values, tzinfo=self.tzinfo).naive
+        ceil = (floor +
+                relativedelta(**{rel_name: count * rel_steps}) +
```

Comment:
@dgilland The error generated over here in travis build is not getting displayed when I do `inv test` in my local tests also. But, when I run `py.test tests/test_zulu.py -k span` all those tests are passing without any error.